src/claimDetail.js

```diff
@@ -59,83 +85,811 @@ function extractData() {
+            What's next
+          </h2>
+          <div style="background-color: #fff; padding: 24px 32px">
+            <ul
+              style="padding-inline-start: 20px; margin-block-start: 0; margin-block-end: 0"
+            >
+              ${whatsNextContent}
+            </ul>
+          </div>
+        </div>`
+      : ""}
+    <div style="margin: 0 20px 8px">${RETURN_TO_TOP_LINK}</div>
+    ${FOOTER_HTML}`;
 
   root.append(newContainer);
+  const accordions = document.querySelectorAll(".accordion-header");
+  accordions.forEach((accordionEl) => {
+    new Accordion(accordionEl);
+  });
 }
 
-function logView(claimType, claimStatus, claimNotes) {
-  const parsedNotes = claimNotes
-    .map((note) => {
-      if (note.includes("Error while processing")) {
-        return "ERR";
-      } else if (note.includes("Private Plan Operations")) {
-        return "PRIV";
-      } else if (note.includes("Disability During Unemployment")) {
-        return "DDU";
-      } else if (note.includes("invalid because you did not meet the wage")) {
-        return "INVAL_WAGE";
-      } else if (
-        note.includes("ineligible because you failed") &&
-        note.includes("Form C10")
-      ) {
-        return "INEL_C10";
-      } else if (
-        note.includes("ineligible because you failed") &&
-        note.includes("Form M10")
-      ) {
-        return "INEL_M10";
-      } else if (
-        note.includes("ineligible because you failed") &&
-        note.includes("additional information and medical certification")
-      ) {
-        return "INEL_5107";
-      } else if (note.includes("A request for information has been mailed")) {
-        return "SENT_REQ";
-      } else if (note.includes("No decision has been made")) {
-        return "14_DAY";
-      } else if (
-        note.includes("Request to Claimant for Information was sent to")
-      ) {
-        return "SENT_C10";
-      } else if (note.includes("Request for Medical Information was sent to")) {
-        return "SENT_Mxx";
-      } else if (note.includes("Request for Wage information was sent to")) {
-        return "SENT_Exx";
-      } else if (
-        note.includes(
-          "Notice of Required Pursuit of Workers' Compensation Claim was sent to"
-        )
-      ) {
-        return "SENT_W10";
-      } else if (
-        note.includes("Request to Claimant for Information was generated")
-      ) {
-        return "GEN_C01";
-      } else if (note.includes("Form W01")) {
-        return "GEN_W01";
-      } else if (note.includes("Form M-01")) {
-        return "GEN_M01";
-      } else if (note.includes("An Employer Statement")) {
-        return "GEN_E01";
-      } else {
-        return note;
-      }
-    })
+function logView(metadata) {
+  const { claimType, claimStatus, claimNotes } = metadata;
+  const parsedNotes = getParsedClaimNotes(claimNotes)
+    .map((note) => note.type)
     .join(";");
 
   logEvent("[DOL_DABI] Viewed Claim Detail page", {
```

Comment:
Might be worth limiting char length of the param values to not exceed 100 just in case (it looks like `object_details` is longer although not necessarily that long, but I don't know how common it is to have a lot there)

Reply:
Hmm yea it's a fair point. I haven't seen the data going over the maximum from analytics which is a good sign. But if it does go over, it will any truncate it to 100, so it feels like the real solution will need to be shortening the notes further to fit more in. I think this may be out of scope here, and we can tackle if we see there's data that's getting truncated.

Reply:
I think I was misremembering some other time in analytics where the truncation caused issues (i think because the param value had curly brackets so  so it was more important to truncate the content inside them) - in this case I'm realizing it's a string value without that so it wouldnt be the same issue!

Reply:
Ah yes that was for a JSON string, which would indeed break when truncated